src/services/services.ts

```diff
@@ -725,7 +725,7 @@ namespace ts {
 
     class SourceMapSourceObject implements SourceMapSource {
         lineMap: number[];
-        constructor (public fileName: string, public text: string, public skipTrivia?: (pos: number) => number) {}
+        constructor(public fileName: string, public text: string, public skipTrivia?: (pos: number) => number) { }

```

Comment:
All these white-space changes are interesting. Is it passing the linter? (Was it before?)